modules/using-cli.adoc

```diff
@@ -0,0 +1,221 @@
+// Module included in the following assemblies:
+//
+// * cli/using-roxctl-cli.adoc
+:_module-type: CONCEPT
+[id="using-cli_{context}"]
+= Using the roxctl CLI
```

Comment:
I'm a little torn on the commands in this section. My first thought was that they should be written as proper procedure modules, but then I realized they're more along the lines of a CLI reference like we have here: https://docs.openshift.com/container-platform/4.7/cli_reference/openshift_cli/developer-cli-commands.html

Though yours has more detailed information and is less of a reference.

Let me think on it just a little more, but it might be fine as is.